src/main/java/org/folio/rmapi/builder/QueriableUrlBuilder.java

```diff
@@ -52,19 +56,20 @@ public String build(){
     parameters.add("count=" + count);
     parameters.add("orderby=" + determineSortValue(sort, q));
 
-    return "vendors?" + String.join("&", parameters);
+    return String.join("&", parameters);
   }
 
   private String determineSortValue(Sort sort, String query) {
     if(sort == null){
-      return query == null ? VENDOR_NAME_PARAMETER : RELEVANCE_PARAMETER;
+      return query == null ? nameParameter : RELEVANCE_PARAMETER;
     }
     switch (sort){
       case RELEVANCE:
         return RELEVANCE_PARAMETER;
       case NAME:
-        return VENDOR_NAME_PARAMETER;
+        return nameParameter;
+      default:
```

Comment:
Here, I think we should change the default to be "relevance" instead of throwing an invalid sort param

Reply:
Here sort is guaranteed to be not null, because there is a null check above,
"default:" part is included just to make java code compile, basically "default:" will only be executed if in the future someone adds more possible Sort enum values

Reply:
yes